content/Solution for common problems - Section 1 - Data - Mikaela Pisani.md

```diff
@@ -0,0 +1,160 @@
+![](https://cdn.pixabay.com/photo/2018/01/08/21/19/idea-3070332_960_720.jpg)
+
+# SOLUTIONS FOR COMMON PROBLEMS IN DATA SCIENCE & MACHINE LEARNING 
+
+## SECTION 1 - DATA
+
+# SOLUTION TO COMMON PROBLEMS
+
+The theory  is not always the same as the reality. When applying data science in practice, several you will have to deal with several problems. Fortunately, you are not alone!, other people might be in a similar situation and there are blogs, slack channels and information around that might help you to resolve your problems. 
```

Comment:
`several you will have to deal with several problems` I think you need to remove the first 'several'